pkg/ddevapp/apptypes_test.go

```diff
@@ -106,11 +106,59 @@ func TestConfigOverrideAction(t *testing.T) {
 		app.PHPVersion = newVersion
 		err = app.WriteConfig()
 		assert.NoError(err)
-		err = app.ConfigFileOverrideAction()
+		err = app.ConfigFileOverrideAction(false)
 		assert.NoError(err)
 		// But with a config that has been written with a specified version, the version should be untouched by
```

Comment:
```suggestion
		// But with a config that has been written with a specified version, the version should be untouched 
```

Reply:
This is intended to read "the version should be untouched by app.ConfigFileOverrideAction()" so it's correct I think.

Reply:
Yes, but the ` app.ConfigFileOverrideAction()` line was removed? Might be a glitch on github diff rendering, didn't look at this locally.

Reply:
```suggestion
		// But with a config that has been written with a specified version, the version should be untouched by
		// app.ConfigFileOverrideAction()
```